docs/nftfair/users/faq.md

```diff
@@ -16,44 +18,43 @@ A user-facing activity site is also created for a Fairdrop event, for participan
 
 ### **1. Can I start a Fairdrop activity to sell or give away NFTs from an existing contract?**
 
-Currently, this platform only supports NFT contract that follows the ERC721 standard.
+Currently, this platform only supports NFT contracts that follow the ERC721 standard.
```

Comment:
We also support ERC1155 now. Besides, to answer this question: yes, you can transfer these NFTs to our vault contract and start the fairdrop.